admin/static/js/pages/prereg-admin-page.js

```diff
@@ -0,0 +1,254 @@
+*/
+var Row = function(params, permission, reviewers) {
+    var self = this;
+
+    self.params = params;
+    self.viewingDraft = ko.observable(false);
+    if (permission === "false") {
+        permission = false;
+    } else {
+        permission = true;
+    }
+    self.adminPermission = ko.observable(permission);
+
+    self.editing = ko.observable(false);
+
+    self.title = params.registration_metadata.q1.value || 'no title';
+    self.fullname = params.initiator.fullname;
+    self.username = params.initiator.emails[0].address;
+    self.initiated = self.formatTime(params.initiated);
+    self.updated = self.formatTime(params.updated);
+    self.status = ko.observable(params.is_pending_approval ? 'pending approval': 'approved');
+
+    self.proofOfPub = new ProofOfPub();
+    self.paymentSent = new PaymentSent();
+    self.notes = new Notes();
+    self.assignee = new Assignee(reviewers); 
+};
+
+
+Row.prototype.highlightRow = function(data, event) {  
+    var row = event.currentTarget;
+    $(row).css("background","#E0EBF3"); 
+};
+
+Row.prototype.unhighlightRow = function(data, event) {
+    var row = event.currentTarget;
+    $(row).css("background",""); 
+};
+
+Row.prototype.formatTime = function(time) {
+    var parsedTime = time.split(".");
+    return parsedTime[0]; 
+};
+
+Row.prototype.goToDraft = function(data, event) {
+    var self = this;
+    if (self.editing() === false) {
+        self.viewingDraft(true);
+        document.location.href = 'drafts/' + self.params.pk + '/';
+    }
+};
+
+/**
+* The view containing all of the rows with draft information.
+*
+* @param  {String}  adminSelector   The class to bind to
+* @param  {String}  user            The current user
+* @param  {List}    reviewers       The prereg reviewers who can be assigned
+*/
+var AdminView = function(adminSelector, user, reviewers) {
+    var self = this;
+
+    self.adminSelector = adminSelector;
+    self.user = user;
+    self.reviewers = reviewers;
+
+    self.getDrafts = $.getJSON.bind(null, "drafts/");
+
+    self.drafts = ko.observable([]);
+    self.loading = ko.observable(true);
+
+    self.sortBy = ko.observable('title');
+    self.sortedDrafts = ko.computed(function() {
+        var row = self.sortBy();
+        return self.drafts().sort(function (left, right) { 
+            var a = deep_value(left, row).toLowerCase();
+            var b = deep_value(right, row).toLowerCase();
+            return a == b ? 0 : 
+                (a < b ? -1 : 1); 
+        });
+    });
+};
+
+AdminView.prototype.init = function() {
+    var self = this;
+
+    var getDrafts = self.getDrafts();
+
+    // create new view model for each row
+    getDrafts.then(function(response) {
+        self.drafts(
+            $.map(response.drafts, function(draft){
+                return new Row(draft, self.user.admin, self.reviewers);
+            })
+        );
+    });
+
+    $.when(getDrafts).then(function() {
```

Comment:
It would be better not to use `$.when` for a single promise, as this is typically used when you need to run a callback after multiple promises are resolved. L213  could just go in the done callback to `getDrafts`. 

You might consider adding a `fail()` callback here as well so the user gets feedback if the request fails.